### ISSUE TITLE
[Feature request] Import or creat Projects from Deck or by standard Nextcloud workflow

### ISSUE THREAD

**Issue by gmmit** [mtierltd/timetracker#150]
It would be really nice if we creat a project and assign it to a deck, that this project is automatically imoprtated into "time tracker" app. In addition, it would be very helpful if I could also include "cards" in the time tracker.

https://docs.nextcloud.com/server/latest/user_manual/en/files/projects.html?highlight=projects
Thanks.

**Comment by okko7** [comment]
I'd like to second that. I like working with Decks to manage my tasks, but also need to keep track of time. Would be fantastic if the two apps could be integrated somehow.

**Comment by GeigerCounter** [comment]
I agree with this and think that it goes well with my feature request: https://github.com/mtierltd/timetracker/issues/186

**Comment by thiswillbeyourgithub** [comment]
Very interested in this as well!

**Comment by user12257** [comment]
I agree, the time tracked could be connected to a task/card from the deck.
